KinConnections/static/vendor/php-email-form/validate.js

```diff
@@ -57,14 +57,14 @@
     })
     .then(response => {
       if( response.ok ) {
-        return response.text()
+        return response.statusText

```

Comment:
@shaniljasani I was able to fix the error you had mentioned. The problem was when the response text was being read by calling `response.text()` the return value was `"OK"` including the `"` so when line 67 was checking for equality it was correctly saying that `OK` and `"OK"` are not equal. The only elegant solution I could think of here was to call `statusText` instead of `text()`

Reply:
Since this is vendor code I tried checking online to see if this was a documented error but couldn't find anything.